### DISCUSSION TITLE
2 clusters are getting formed in ETCD

### DISCUSSION THREAD

**Discussion by rahulbapumore** [etcd-io/etcd#15916] [Q&A]
Hi @ahrtr 
We were executing some steps on ETCD cluster and we ended up in 2 cluster situation in etcd.
Could you please guide us how to keep only one cluster and remove other one?
In one of the cluster we have unstarted member, could you guide on how to make it started?

Thanks in advance

**Comment by rahulbapumore** [comment]
@chaochn47 Could u please help?

**Reply by chaochn47** [reply]
Could you please include more information for us to better assist you? 

1. `etcdctl --endpoints <...> endpoint status -w table`? 
2. etcd process command line arguments, specifically clustering section. https://etcd.io/docs/v3.5/op-guide/configuration/#clustering

**Comment by rahulbapumore** [comment]
![image](https://github.com/etcd-io/etcd/assets/25705155/f7764898-ec49-490d-b8db-87556d1c6413)

**Comment by rahulbapumore** [comment]
See here I am running etcdctl member list command 2 times,
1 time it gives output with cluster having 3 members,
and second time it gives cluster with 1 member.
pod 0 is joining new cluster...
how do we delete second cluster which is having 1 member
@chaochn47

**Comment by rahulbapumore** [comment]
![image](https://github.com/etcd-io/etcd/assets/25705155/c4bbeef1-5bc5-4573-86cc-4895502732b9)

When I try to remove that member which alone in another cluster, it does not allow me to do it..
it says not enough started members

**Comment by rahulbapumore** [comment]
@chaochn47 Please help

**Comment by rahulbapumore** [comment]
`bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | fa78de538c75d45d | 3.5.3 | 266 kB | false | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | 25edf1c89ac5d942 | 3.5.3 | 266 kB | true | false | 6 | 262 | 262 | | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ bash-4.4$ unset ETCDCTL_ENDPOINTS;etcdctl --endpoints=https://etcd.zmorrah:2379 endpoint status -w table +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | ENDPOINT | ID | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS | +-----------------------------------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+ | https://etcd.zmorrah:2379 | c4f74813ba82f635 | 3.5.3 | 20 kB | true | false | 2 | 10 | 10 | | +-------------------------------`

**Comment by rahulbapumore** [comment]
multiple time I ran etcdctl endpoint status command,
this is the output which is shared above

**Comment by rahulbapumore** [comment]
@ahrtr @chaochn47

**Comment by liangyuanpeng** [comment]
Any step can reproduce it?

**Comment by ahrtr** [comment]
PLease try to run [etcd-diagnosis](https://github.com/ahrtr/etcd-diagnosis) and provide the report
